core/templates/pages/exploration-player-page/services/exploration-player-state.service.ts

```diff
@@ -160,12 +160,10 @@ export class ExplorationPlayerStateService {
     this.playthroughService.initSession(
       this.explorationId, returnDict.version,
       returnDict.record_playthrough_probability);
-    this.playerCorrectnessFeedbackEnabledService.init(
-      returnDict.correctness_feedback_enabled);
+    this.playerCorrectnessFeedbackEnabledService.init(true);
```

Comment:
Do we still need these calls and this service?

Reply:
Though we have removed correctness feedback from the backend completely, it makes sense to keep this in the frontend to provide more context, and it's being used in multiple places.

Reply:
No, I disagree. If it's not modifiable then why do we need to have an entire service for maintaining whether we show it or not? We don't have similar services for all the other small features in the exploration player (unless they are configurable or we need to maintain state in some way).

Reply:
understood. yes, this service is totally unnecessary now will remove it.